doc/nrf/libraries/nrf_rpc/nrf_rpc_uart.rst

```diff
@@ -10,29 +10,43 @@ nRF RPC UART transport
 The nRF RPC UART transport allows you to use the :ref:`nrf_rpc` library to execute procedures on a remote processor that is connected with the local processor using the UART interface.
 This transport is used in the :ref:`nrf_rpc_ps_samples`.
 
+Configuration
+*************
+
+Set :kconfig:option:`CONFIG_NRF_RPC_UART_TRANSPORT` Kconfig option to enable the nRF RPC UART transport.
+
+Define the ``nordic,rpc-uart`` property in a devicetree overlay file to select the UART peripheral used by the API serialization libraries within |NCS|, such as :ref:`ble_rpc` or :ref:`nfc_rpc`.
+
+.. code-block:: devicetree
+
+   / {
+      chosen {
+         nordic,rpc-uart = &uart1;
+      };
+   };
+
 Frame encoding
 **************
 
 .. note::
 
    The current frame format is experimental and is a subject to change.
 
-An nRF RPC packet that is sent using the nRF RPC UART transport is encoded within a frame whose format resembles the one used by the HDLC protocol.
-
-Each frame shall start and end with the **delimiter** octet (``0b01111110`` = ``0x7e``).
-Each two subsequent frames may be separated by more than one delimiter octet.
+An nRF RPC packet that is sent using the nRF RPC UART transport is encoded within a frame whose format resembles the one used by the HDLC protocol:
 
-Each byte of the nRF RPC packet shall be encoded according to the following rules:
+1. Each frame shall start and end with the **delimiter** octet (``0b01111110`` = ``0x7e``).
+#. Each two subsequent frames may be separated by more than one delimiter octet.
+#. Each byte of the nRF RPC packet shall be encoded according to the following rules:
```

Comment:
```suggestion
* Each frame shall start and end with the **delimiter** octet (``0b01111110`` = ``0x7e``).
* Each two subsequent frames may be separated by more than one delimiter octet.
* Each byte of the nRF RPC packet shall be encoded according to the following rules:
```
These aren't something that happens sequentially, so a numbered list isn't correct here.